CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [4.0.0] - 2020-02-05
+
```

Comment:
Adding a bolder breaking change warning could be better. Something like: (not final wording, to give an idea only)

```suggestion
**BREAKING CHANGE**: Legacy domain support on PSaaS is removed in this release. If you depend on this feature, you should keep on using an earlier version. This change doesn't introduce breaking changes to our cloud customers.
```